.github/wiki/Getting-Started.md

```diff
@@ -44,6 +44,25 @@ cd() {
 check_directory_for_new_repository
 ```
 
+By @TheSast
+
+A fish adaptation of the previous script to put in your `$XDG_CONFIG_HOME/fish/functions/cd.fish`:
+```fish
+function cd -w='cd'
+  builtin cd $argv || return
+  check_directory_for_new_repository
+end
+
+function check_directory_for_new_repository
+  set current_repository (git rev-parse --show-toplevel 2> /dev/null)
+  if [ "$current_repository" ] && \
+    [ "$current_repository" != "$last_repository" ]
+    onefetch
+  end
+  set -x last_repository $current_repository
```

Comment:
```suggestion
  set -gx last_repository $current_repository
```

When I tested locally, it seems to always run `onefetch` when navigating inside a repository.

To fix this issue, I added the -g flag to the set command to mark last_repository as a global variable, so its value is remembered between function calls

Reply:
Thanks for catching the typo!